.zuul.yaml

```
@@ -3,14 +3,19 @@
     name: functional-autoscaling-tests-osp18
     parent: telemetry-operator-multinode-autoscaling
     description: |
-      Run the autoscaling functional test on osp18+patched version of aodh and heat.
+      Run job on osp18+patched version of aodh and heat for
+      autoscaling and metrics functional.
```

```suggestion
      autoscaling and metrics functional tests.
```
Seems like there is something missing to the sentence, adding "tests" or something similar would sound better IMO